.packit.yaml

```diff
@@ -140,9 +140,45 @@ jobs:
       - centos-stream-9-aarch64
     skip_build: true
     manual_trigger: true
+    env: { IP_FAMILY: ipv4 }
     labels:
       - sanity
     tf_extra_params:
       test:
         tmt:
           name: sanity
+
+  ###############################################################################################
+  ##################################    IPv6/dual jobs   ########################################
```

Comment:
I wonder if we should merge one acceptance job, which will trigger also IPv6 and dual configuration instead of only IPv4